src/client/sandbox/xhr.js

```diff
@@ -73,6 +81,31 @@ export default class XhrSandbox extends SandboxBase {
             nativeMethods.xhrRemoveEventListener.call(this, 'readystatechange', syncCookieWithClient);
         };
 
+        const xmlHttpRequestWrapper = function () {
+            const xhr = new nativeMethods.XMLHttpRequest();
+
+            nativeMethods.xhrAddEventListener.call(xhr, 'readystatechange', emitXhrCompletedEvent);
+            nativeMethods.xhrAddEventListener.call(xhr, 'readystatechange', syncCookieWithClient);
+
+            return xhr;
+        };
+
+        for (const prop of ['UNSENT', 'OPENED', 'HEADERS_RECEIVED', 'LOADING', 'DONE']) {
```

Comment:
Store as `const XHR_READY_STATES = ['UNSENT', 'OPENED', 'HEADERS_RECEIVED', 'LOADING', 'DONE']

 and rewrite as

`for (const readyState of XML_HTTP_REQUEST_READY_STATES)...`